.github/workflows/repository_messaging.yml

```diff
@@ -9,6 +9,7 @@ on:
 jobs:
   holiday_message:
     runs-on: ubuntu-latest
+    if: false
```

Comment:
Perhaps worth adding a comment here saying something like "If situation XXX occurs, do YYY to turn this on." (It doesn't seem obvious to me as a possible future contributor that the natural thing to do is to drop this line altogether.)

Reply:
Good suggestion--done.